packages/browser/test/tracing/browserTracingIntegration.test.ts

```diff
@@ -226,29 +236,6 @@ describe('browserTracingIntegration', () => {
     });
   });
 
-  it("trims pageload transactions to the max duration of the transaction's children", async () => {
```

Comment:
I deleted this test because switching to the vitest timers kept it failing - but does this behaviour even apply anymore with the new changes?

Maybe @mydea you know what is happening here?